.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,8 +17,17 @@ Example: GRW-123 / Feature / Awesome new thing
 <!-- Why are these changes made? -->
 
 
-_Referenced ticket(s): []_
-<!-- If there is a Jira issue, add the id between brackets, and a link to that issue will be created. -->
+<!-- If there is a Jira issue, add the key (e.g. GRW-123) between brackets below, and a link to that issue will automaticaly be created. -->
+**Referenced ticket(s): []**
+
 
 
-<!-- Finally, if that makes sense, add screenshots and/or screen recordings below, preferably with headlines and/or descriptions -->
+<!-- If it makes sense, add screenshots and/or screen recordings below, preferably with headlines and/or descriptions -->
+<!--
+## Screenshots / recordings 
+-->
+
+<!-- Finally, you can create a review app on Heroku if you think that will be helpful to the reviewers -->
```

Comment:
Great! 🙌 Just a suggestion to add that review apps are great if we want feedback from design before we merge to staging :)
```suggestion
<!-- Finally, you can create a review app on Heroku if you think that will be helpful to the reviewers and getting input from the design team before merging -->
```

Reply:
Yes, I wrote about this on the new [Review Apps page](https://www.notion.so/hedviginsurance/Review-apps-886358fb864d42f4b7dd66d49fd8aef4) in Notion, but it might be worth mentioning here as well unless it becomes too cluttered.